test/wallet-test-auctions.js

```diff
@@ -0,0 +1,184 @@
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+/* eslint no-return-assign: "off" */
+
+'use strict';
+
+const assert = require('./util/assert');
+const Chain = require('../lib/blockchain/chain');
+const WorkerPool = require('../lib/workers/workerpool');
+const Miner = require('../lib/mining/miner');
+const WalletDB = require('../lib/wallet/walletdb');
+const Network = require('../lib/protocol/network');
+const rules = require('../lib/covenants/rules');
+
+const network = Network.get('regtest');
+const NAME1 = rules.grindName(10, 20, network);
+
+const workers = new WorkerPool({
+  enabled: false
+});
+
+network.coinbaseMaturity = 1;
+
+const chain = new Chain({
+  memory: true,
+  network,
+  workers
+});
+
+const miner = new Miner({
+  chain,
+  workers
+});
+
+const cpu = miner.cpu;
+
+const wdb = new WalletDB({
+  network: network,
+  workers: workers
+});
+
+describe('Wallet Auction', function() {
```

Comment:
I think that once we move this test to `auctions-test.js`, we should call it something like 'Test OPEN' and have it run after the 'Vickrey Auction' describe. What do you think?

Reply:
Resolved out of band.